Makefile

```diff
@@ -1,4 +1,5 @@
 
+BROWSERIFY := node_modules/.bin/browserify
```

Comment:
No clue, I can repro it in my other local repos, but not in my local Mocha repo. Weird.

I also prefer the explicit path just because I keep global installations of some binaries (e.g. ESLint) installed for editor integrations, etc. and like to be sure there's nothing funky going on, like a missing local dependency triggering errors because it's falling back on a global installation, which is a heinous thing to debug. More verbose for fewer terrible things to debug, but maybe that's crossed over into paranoia territory.

Anyway, it's working locally so I don't mind either way